src/DockApp.cpp

```diff
@@ -52,7 +52,7 @@ void CDockApp::ReadyToRun()
 
 	mwDock->MoveTo( x, y );
 	mwDock->ResizeTo( xw, yw );
-	mwDock->SetSizeLimits( 48, 200, 100, r.bottom - 4 ); 
+	mwDock->SetSizeLimits( 100, 200, 200, r.bottom - 4 );
```

Comment:
I don't think there's a valid reason to limit the max width to 200px. If the user wants to make it wider, why not?
Just set it to 10000, I'd say.

Reply:
OK, wait a minute.